### PR TITLE
JSDK-2312, JSDK-2303

### PR DESCRIPTION
**Pull request by manjeshbhargav**
@syerrapragada 

* JSDK-2312 - Support unified plan on Chrome 72+
* JSDK-2303 - Convert audioLevel range to 0-32767

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,32 @@
...
 
+2.0.0-beta8 (in progress)
+=========================
+
+New Features
+------------
+
+- twilio-video.js will now use the Unified Plan SDP format where available.
+  Google Chrome, starting from version 72, has and Safari, starting from version
+  12.1, will enable Unified Plan as the default SDP format. We highly recommend
```
Comment (syerrapragada) [inline CHANGELOG.md]: 12.1 does not have a unified plan. Also, we have already announced support for the unified plan on Safari in 2.0.0-beta6.

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,32 @@
 For 1.x changes, go [here](https://github.com/twilio/twilio-video.js/blob/support-1.x/CHANGELOG.md).
 
+2.0.0-beta8 (in progress)
```
Comment (syerrapragada) [inline CHANGELOG.md]: 2.0.0-beta7 is still in progress. So this wont be part of that I guess?

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,5 +1,32 @@
...
+=========================
+
+New Features
+------------
+
+- twilio-video.js will now use the Unified Plan SDP format where available.
+  Google Chrome, starting from version 72, has and Safari, starting from version
+  12.1, will enable Unified Plan as the default SDP format. We highly recommend
+  that you upgrade your twilio-video.js dependency to this version so that your
+  application continues to work on the above mentioned browser versions.
```
Comment (syerrapragada) [inline CHANGELOG.md]: On chrome, users are not really affected but just that they won't be using the default format?

**Comment by syerrapragada** [comment]
> @manjeshbhargav I did not see changes related to `getSdpFormat`? Shouldn't we also add changes to find the default SDP format on Chrome?

Ah! Nevermind I just saw the other PR. Changes look good except few questions related to changelog changes.

File: CHANGELOG.md (hunk 2 above)
Comment (manjeshbhargav) [reply CHANGELOG.md]: @syerrapragada Yes. We will release this after releasing beta7.

File: CHANGELOG.md (hunk 3 above)
Comment (manjeshbhargav) [reply CHANGELOG.md]: @syerrapragada On chrome, they will be using unified plan, if it is the default format.

**Comment by manjeshbhargav** [comment]
Closing this PR and continuing in #592 .

**Pull request closed without merging**